nservicebus/operations/nservicebus-analyzer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: NServiceBus Analyzer
+summary: How to use the NServiceBus analyzer to avoid missing awaits
+component: Core
+versions: '[7,]'
```

Comment:
We're also shipping in 6.5. How do we handle that in the docs?

Reply:
I've started with v7 as the doco was written before we completed the backporting. I have changed the version span to start from v6 now.